test/unit/controllers/CommentController.test.js

```diff
@@ -216,4 +216,42 @@ describe('CommentController', function () {
       })
     })
   })
+
+  describe.only('createBatchFromEmailForm', () => {
```

Comment:
btw, in dev, you can do `grunt autotest --file=test/unit/controllers/CommentController.test.js` to just re-run that one file